presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/eventcreation/detail/EventCreationDetailScreen.kt

```diff
@@ -117,8 +136,10 @@ fun EventCreationDetailScreen(
                     modifier = Modifier
                         .fillMaxWidth()
                         .padding(top = 16.dp, bottom = 24.dp),
-                    date = state.date,
-                    textColor = Gray60,
+                    date = state.date?.let {
+                        LocalDateUtil.format(LocalDateUtil.parseLongToLocalDateTime(it), "yy/MM/dd")
+                    } ?: "",
```

Comment:
```suggestion
                    }.orEmpty(),
```

최근에 알게 됙건데 이런 방법도 있더라구

Reply:
헉 좋다 반영할래.

Reply:
ab6d975204105bd032c9a0f5a1070457f1ed57c6